kanselarij-vlaanderen/frontend-kaleidos
# Fixes KAS-1777 remove ki-min icoon

## What has been done
Op basis van KAS-1777 hebben we een icoontje met een verkeerde viewbox verwijderd (ki-min). Dit omdat het al bestond in de vorm van ki-remove. We hebben nagekeken of het icoontje gebruikt werd in de code base en dit bleek niet het geval te zijn. 

## Future work based on this PR
* We hebben wel gemerkt dat er nog 2 icon fonts ingeladen worden: het ene icon font is het oude icon font van web universum en het andere icon font is het nieuwe icon font gegenereerd met het script dat in de root van de app directory staat. We moeten wegmigreren van het web universum font, dit houdt concreet in dat we alle instanties van het gebruik van dit font moeten vervangen door het nieuwe font. Dit houdt dan ook in dat elk icoontje dat gebruikt wordt beschikbaar moet zijn in het nieuwe font, daar is dus nog wat design werk aan verbonden om dit na te kijken.

* In deze commit zie je dat er door 1 file te verwijderen en een nieuw icon font te genereren er 9 files worden aangepast. We moeten generated files niet committen, maar omdat we een eenvoudige test commit wouden doen hebben we het gelaten zoals het was.

## How to test
1. Go to the `/styleguide` url 
2. Go to the icons documentations page
3. Observe cleaner page without the viewbox mistake (view screenshots in [KAS-1777](https://kanselarij.atlassian.net/browse/KAS-1777))

## Extra opmerking
Dit is een zeer minieme change, Johan is alle changes aan het opdelen en bekijken of we deze minieme changes niet kunnen bundelen. We hebben nu toch besloten om hier een PR van te maken om mij te leren hoe in dit project een PR dient gemaakt te worden. Als jullie opmerkingen hebben over deze PR, graag feedback hierover.
